src/execution_log.rs

```diff
@@ -179,6 +183,7 @@ pub trait ExecutionLog: Sync + Send {
 }
 
 #[derive(Debug)]
+/// A default implementation of an execution log.
 pub struct DefaultExecutionLog<H: BlockHash> {
```

Comment:
As this is now public, I would rename it, `DefaultExecutionLog` is not a most fortunate name.
I'd propose `TransactionExecutionLog`, but I am open to other ideas.

Reply:
Done.